public/app/features/playlist/PlaylistPage.tsx

```diff
@@ -73,6 +87,23 @@ export const PlaylistPage: FC<Props> = ({ navModel }) => {
           />
         )}
         {content}
+        {playlistItemToDelete && (
+          <ConfirmModal
+            title={playlistItemToDelete.name}
+            confirmText="delete"
+            body={`Are you sure you want to delete '${playlistItemToDelete.name}' playlist?`}
+            onConfirm={() => {
+              deletePlaylist(playlistItemToDelete.id).finally(() => {
+                setForcePlaylistsFetch(forcePlaylistsFetch + 1);
+                setPlaylistToDelete(undefined);
+              });
+            }}
+            isOpen={Boolean(playlistToDelete)}
+            onDismiss={() => {
+              setPlaylistToDelete(undefined);
+            }}
```

Comment:
nitpick: we should try to avoid anonymous functions (arrow functions =>) as props because they will be recreated on each render.